app/src/components/Header/Header.tsx

```diff
@@ -11,23 +12,38 @@ import MobileNav from './MobileNav'
 import { ConnectWalletButton, Navbar } from './styled'
 import { useRouter } from 'next/router'
 import { activeMenuItems } from './menuItems'
+import { setCookies, getCookie } from 'cookies-next'
+import injected from 'config/Walletconnectors'
 
 const Header = () => {
-  const { account: address, deactivate } = useWeb3()
+  const { account: address, deactivate, activate } = useWeb3()
   const { t } = useTranslation('header')
   const [showWalletModal, setWalletModal] = useState(false)
   const [showMobileNav, setModalNav] = useState(false)
   const { pathname } = useRouter()
   const navMenu = activeMenuItems(pathname)
 
+  const IS_WALLET_CONNECTED = 'is_wallet_connected'
+
   async function disconnect() {
     try {
       deactivate()
+      setCookies(IS_WALLET_CONNECTED, false)
```

Comment:
We should set the wallet address in the cookie instead: account = wallet address